static-files/webpack.config.js

```diff
@@ -50,14 +50,9 @@ module.exports = (env = {}, options = {}) => {
 
       // NOTE: when using mockup with 'yarn dev', only static.js will be served
       if (shouldBuildStaticSite) {
-        entries.static = [
-          'babel-polyfill',
-          'whatwg-fetch',
-          './source/static.js'
-        ];
+        entries.static = ['whatwg-fetch', './source/static.js'];
```

Comment:
Why no polyfill?

Reply:
There is, but it's based on usage. `"useBuiltIns": "usage"`.